linkis-commons/linkis-storage/src/main/scala/org/apache/linkis/storage/resultset/ResultSetReader.scala

```diff
@@ -64,6 +65,7 @@ object ResultSetReader {
 
   def getTableResultReader(res: String): ResultSetReader[TableMetaData, TableRecord] = {
     val rsFactory = ResultSetFactory.getInstance
+    logger.info(s"getTableResultReader, file path:${res}")
```

Comment:
remove

Reply:
need add log before ` fs.init(null)` 
logger.info("Try to init Fs with path:"+ resPath.getPath)